extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ClassConfigPropertiesUtil.java

```diff
@@ -349,10 +352,18 @@ private static ResultHandle populateConfigObject(ClassLoader classLoader, ClassI
             if (superClassDotName.equals(DotNames.OBJECT)) {
                 break;
             }
-            currentClassInHierarchy = applicationIndex.getClassByName(superClassDotName);
-            if (currentClassInHierarchy == null) {
+
+            ClassInfo newCurrentClassInHierarchy = applicationIndex.getClassByName(superClassDotName);
+            if (newCurrentClassInHierarchy == null) {
+                if (!superClassDotName.toString().startsWith("java")) {
```

Comment:
should it be `java.` like in ?https://github.com/quarkusio/quarkus/blob/4fb30927b9ffc31cb15d86185e243d7b4cacb823/core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/ReflectiveHierarchyBuildItem.java#L73

Reply:
Good idea

Reply:
Fixed